pagopa/pn-frontend
# Hide CTA buttons in DigitalContactElem during PEC verification state

## Short description
Hide CTA buttons in DigitalContactElem during PEC verification state.

## List of changes proposed in this pull request
- Added new prop in DigitalContactElem component to toggle buttons show / hide
- Added new tests

## How to test
Add a new PEC in contacts page, enter verification code. You'll see label "Verifica in corso..." and buttons like "Modifica" and "Elmina" should not be visible.
